### PR TITLE
Update README.md in healthcareapi

### PR DESCRIPTION
**Pull request by simanamaria**
Updated the grammar in the README.md file in [healthcareapis]

(https://github.com/Azure/azure-sdk-for-java/tree/c1273c2b91390fc2496d8ac1c05dd185f58d746b/sdk/healthcareapis).

### CODE REVIEW

**Review by weidongxu-microsoft — APPROVED** [review]
Any background?

The readme is via a template.

**Comment by weidongxu-microsoft** [comment]
@simanamaria 

If you have improvement on the readme, we can get service to review and approve, and we can update template to make future doc better.

**Comment by simanamaria** [comment]
•	Line 44 – avoid unnecessary repetition: https://www.scribbr.com/academic-writing/repetition-redundancy “Avoid repetition of particular sounds or words (e.g. “Several shelves sheltered similar sets of shells”)”
•	Line 48 – correct ( https://www.vocabulary.com/dictionary/correct ) in this instance is an adjective (ex” the correct answer”) and adjectives are usually placed before the nouns (https://www.bbc.co.uk/worldservice/learningenglish/grammar/learnit/learnitv133.shtml ) and are used to describe nouns, but “configure” is a verb (https://dictionary.cambridge.org/dictionary/english/configure ) which should be exchanged for the noun “configuration” (https://dictionary.cambridge.org/dictionary/english/configuration) 
•	Line 48 – “the”( http://www.butte.edu/departments/cas/tipsheets/grammar/articles.html )
o	No article is used before general nouns (ex. trees)
o	The article “the” is used before nouns that refer to specific things, in this case the environment variables AZURE_CLENT_ID, AZURE_TENANT_ID etc. (these are not abstract or general nouns, but specific)
•	Line 54 - https://dictionary.cambridge.org/grammar/british-grammar/noun-phrases-order the order in a noun phrase is article  adjective  noun as modifier  head (https://en.wiktionary.org/wiki/noun_modifier ) the noun modifier in this case is the specific variable AZURE_SUBSCRIPTION_ID and the environment variable is the head.
•	Line 56 - The article “the” is used before nouns that refer to specific things, and there are 2 options here:
o	Either following (https://dictionary.cambridge.org/dictionary/english/following ) is an adjective – in this case the structure should be “the following code”
o	Or it is used as a verb – in which case the structure should be “by following the code”
o	Line 56 – another option - stylistic preference - Replace “by” with “using the” – (https://www.quora.com/When-should-I-use-by-and-when-should-I-use-using-in-a-sentence (ex: I tried to call you by phone. When my phone was broken, I tried to call you using a friend’s phone.) )

**Comment by simanamaria** [comment]
@weidongxu-microsoft these are the changes I would make, documented with resources. These changes can be applied to the https://github.com/Azure/autorest.java/blob/main/fluentgen/src/main/resources/Readme.txt as well. (on line 56 there are more than one option in order to improve the readability of the text, and the last option is my preference)

**Comment by weidongxu-microsoft** [comment]
@simanamaria 

Thank you for the contribution, and detailed explanation. We will review it closely, and update the template as well.

File: sdk/healthcareapis/azure-resourcemanager-healthcareapis/README.md [hunk 1]
```diff
@@ -41,19 +41,19 @@ Various documentation is available to help you get started
...
 
-By default, Azure Active Directory token authentication depends on correct configure of following environment variables.
+By default, Azure Active Directory token authentication depends on correct configuration of the following environment variables.
 
 - `AZURE_CLIENT_ID` for Azure client ID.
 - `AZURE_TENANT_ID` for Azure tenant ID.
 - `AZURE_CLIENT_SECRET` or `AZURE_CLIENT_CERTIFICATE_PATH` for client secret or client certificate.
 
-In addition, Azure subscription ID can be configured via environment variable `AZURE_SUBSCRIPTION_ID`.
+In addition, Azure subscription ID can be configured via `AZURE_SUBSCRIPTION_ID` environment variable .
```
Comment (weidongxu-microsoft) [inline sdk/healthcareapis/azure-resourcemanager-healthcareapis/README.md:54]:
```suggestion
In addition, Azure subscription ID can be configured via `AZURE_SUBSCRIPTION_ID` environment variable.
```

**Review by weidongxu-microsoft — APPROVED** [review]
Thanks @simanamaria. It indeed makes the doc easier to read (and being correct) this way.

PR to the template https://github.com/Azure/autorest.java/pull/1761, it would be enabled next month, and will apply to all new management-plane libs.

**Comment by simanamaria** [comment]
Ok, Thank you!

File: sdk/healthcareapis/azure-resourcemanager-healthcareapis/README.md (hunk 1 above)
Comment (weidongxu-microsoft) [reply sdk/healthcareapis/azure-resourcemanager-healthcareapis/README.md]: update it to retrigger the CI, for known cla issue.

**Pull request merged**